terraform/paas/variables.tf

```diff
@@ -43,10 +43,17 @@ variable "paas_linked_services" {
   , "get-into-teaching-api-dev-pg-common-svc"]
 }
 
+variable "paas_monitoring_space" {
+  default = "sandbox"
+}
+
 variable "paas_space" {
   default = "sandbox"
 }
 
+variable "paas_monitoring_app" {
+  default = "prometheus-dev-get-into-teaching"
```

Comment:
If we always expect to pass a value, we can remove the default

Reply:
removed